webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue

```diff
@@ -20,7 +20,12 @@
         <v-list-item-title class="text-wrap">
           <template v-if="enabledDigestLabel">
             <v-chip class="ma-2" color="primary">
-              {{ enabledDigestLabel }}
+               <span v-if="isMobile" class="mobile-chip-ellipsis">
+                  {{ enabledDigestLabel }}
+              </span>
+              <span v-else>
+                {{ enabledDigestLabel }}
+              </span>
```

Comment:
Following https://vuejs.org/v2/guide/class-and-style.html , this can be replacesd with
```suggestion
               <span :class="{ mobile-chip-ellipsis : isMobile }">
                  {{ enabledDigestLabel }}
              </span>
```